src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,15 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Excavator excavator = new Excavator();
```

Comment:
```suggestion
        Machine excavator = new Excavator();
```
хорошою практикою вважається оголошення більш абстракного посилання для змінною (абстрактний клас / інтерфейс). Це потрібно для того, що можна було легко підміняти імплементації